doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -719,17 +720,19 @@ the toplevel, and using them in source files is discouraged.
 
 .. cmd:: Add ML Path @string
 
-   This command adds the path :n:`@string` to the current OCaml
-   loadpath (cf. :cmd:`Declare ML Module`).
-
+   This command is equivalent to the command line option :n:`-I
+   @string` and adds the path :n:`@string` to the current OCaml
+   loadpath (cf. :cmd:`Declare ML Module`). It is available for
+   convenience, for instance for use in an interactive session, and it
+   is not exported to compiled files. For separation of concerns with
+   respect to the relocability of files, it is recommended to use
+   :n:`-I @string`.
```

Comment:
```suggestion
   Equivalent to the :ref:`command line option <command-line-options>`
   :n:`-I @string`.  Adds the path :n:`@string` to the current OCaml
   loadpath (cf. :cmd:`Declare ML Module`). It is for
   convenience, such as for use in an interactive session, and it
   is not exported to compiled files. For separation of concerns with
   respect to the relocability of files, we recommend using
   :n:`-I @string`.
```

Phrases such as "This command is" are often omitted in manuals rather than repeating them hundreds of times.

Note that a newline in :n: is preserved in the HTML output, which we don't want.

Some of the suggestions are better readability.